src/test/javascript/spec/component/text-exercise/text-exercise.component.spec.ts

```diff
@@ -60,4 +63,24 @@ describe('TextExercise Management Component', () => {
         // THEN
         expect(courseExerciseService.findAllTextExercisesForCourse).toHaveBeenCalled();
     });
+
+    describe('TextExercise Search Exercises', () => {
+        it('Should show all exercises', () => {
+            // WHEN
+            comp.exerciseFilter = new ExerciseFilter('EXT', '', 'text');
+
+            // THEN
+            expect(comp.textExercises.length).toEqual(1);
+            expect(comp.filteredTextExercises.length).toEqual(1);
```

Comment:
From my understanding we should use `toHaveLength` function, as in:
Same for other tests.
```ts
expect(comp.textExercises).toHaveLength(1);
```